### PR TITLE
to_flop.xml: Metadata cleanings

### PR DESCRIPTION
**Pull request by ArcadeShadow**
- Lowercase on descriptive words like "Format", "Alt", "Hacked", "Version", etc.
- Uppercase on abbreviature like "to7" and "to8"
- Correct description "Le Temps d'Une Historie" to "Le Temps d'une Histoire"

### CODE REVIEW

File: hash/to_flop.xml [hunk 1]
```diff
@@ -14171,7 +14171,7 @@ Thanks to DCMOTO (http://dcmoto.free.fr) for info!
 
 	<!-- can also be found in mo6_flop.xml -->
 	<software name="traitbzzb" cloneof="traitbzz">
-		<description>Traitement Bzzz (Compilation Top Arcade)</description>
+		<description>Traitement Bzzz (compilation Top Arcade)</description>
```
Comment (cuavas) [inline hash/to_flop.xml]: I think “Compilation Top Arcade” is the literal name of a collection this was extracted from.

File: hash/to_flop.xml [hunk 2]
```diff
@@ -14317,7 +14317,7 @@ Thanks to DCMOTO (http://dcmoto.free.fr) for info!
 	</software>
 
 	<software name="trivialpb" cloneof="trivialp">
-		<description>Trivial Pursuit (Compilation Top Adventure)</description>
+		<description>Trivial Pursuit (compilation Top Adventure)</description>
```
Comment (cuavas) [inline hash/to_flop.xml]: Same with this one.

Comment (ArcadeShadow) [reply hash/to_flop.xml]: I didn't see the "Compilation" word on the box (for the Top Arcade) neither inside (for the Top Adventure), that's why I removed it.
image
image

Comment (cuavas) [reply hash/to_flop.xml]: Top Adventure and Top Arcade should be turned into multi-part items containing the constituent disks anyway.

Comment (ArcadeShadow) [reply hash/to_flop.xml]: I totally agree, I'll do it next time.

**Pull request merged**